### DISCUSSION TITLE
HTTP error code 404  showing up repeatedly  in the logs

### DISCUSSION THREAD

**Discussion by scottw12** [vanjarosoftware/Vanjaro.Platform#1063] [Q&A] [answered]
I keep getting this log entry over and over. Anyone else have this pop up? 

HTTP Error Code 404 Page Not Found
TabId PortalAlias dev.jcesba.org OriginalUrl /Portals/0/ Url https://
TabId:
PortalAlias:dev.jcesba.org
OriginalUrl:/Portals/0/
Url:https://dev.jcesba.org/Portals/0/
UserAgent:Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.114 Safari/537.36
HostAddress:xxx.xxx.xxx.xxx
HostName:xxx.xxx.xxx.xxx
Server Name: NBBIT02

**Comment by meetmandeep** [comment]
I haven't seen that in particular. Are able to reproduce it? For example, does it happen when you view a certain page or setting?

**Comment by scottw12** [comment, marked as answer]
Odd things are even happening when no one is in editing the site...The is a dev site not readily known, but it is on the net. I am now suspecting some kind of hacking going on as I am seeing some very weird log entries along with the 404 error reported above.


Like this one:
Page Load Exception
AbsoluteURL /default.aspx DefaultDataProvider DotNetNuke.Data.SqlData
AbsoluteURL:/default.aspx
DefaultDataProvider:DotNetNuke.Data.SqlDataProvider, DotNetNuke
ExceptionGUID:fad65c43-fc5a-4ab8-a281-7ea3e2b9e873
AssemblyVersion:9.7.2
PortalId:-1
UserId:-1
TabId:-1
RawUrl:/

Referrer:
UserAgent:Expanse, a Palo Alto Networks company, searches across the global IPv4 space multiple times per day to identify customers&amp;#39; presences on the Internet. If you would like to be excluded from our scans, please send IP addresses/domains to: scaninfo@paloalt

ExceptionHash:1FGHRi1nJ0mTVlqPSb2wjJ/9hTQ=
Message:Object reference not set to an instance of an object.
StackTrace:
InnerMessage:Object reference not set to an instance of an object.
InnerStackTrace:

I would be more than happy to give you the access needed to track this down

**Reply by meetmandeep** [reply]
That explains it. It looks like a bot (do a Google Search on "Expanse UserAgent Palo Alto") is randomly hitting part of the URLs for sniffing.
